src/Recordings.cpp

```diff
@@ -358,6 +377,22 @@ bool Recordings::UpdatePvrRecording(const tinyxml2::XMLNode* pRecordingNode, kod
       }
     }
   }
+  if (m_settings.m_showRoot)
+  {
+    const std::string original = tag.GetDirectory();
+    std::string root = "Default";
```

Comment:
Constant instead of string literal?

Reply:
I don't know what you mean so I removed const

Reply:
Apologies. I meant the the line with the “Default”.

The const was perfectly fine where it was and correct.

Reply:
@phunkyfish std::string root  changes and can't be a constant

Reply:
Oh, right, my mistake.